data/virtualization/autoyast/host_12.xml.ep

```diff
@@ -119,24 +119,15 @@
         <bridge_stp>off</bridge_stp>
         <startmode>auto</startmode>
       </interface>
-      <interface>
-        <device>eth0</device>
-        <bootproto>none</bootproto>
-        <startmode>hotplug</startmode>
-      </interface>
-      <interface>
-        <device>eth1</device>
-        <bootproto>none</bootproto>
-        <startmode>hotplug</startmode>
-      </interface>
     </interfaces>
   </networking>
   <partitioning config:type="list">
     <drive>
-      % if ($check_var->('IPMI_HW', 'supermicro')) {
-      <device>/dev/sda</device>
-      % }
-      <disklabel>msdos</disklabel>
+  % my $wwn = {'lipa' => 'wwn-0x5002538c4002f8bd', briza => 'wwn-0x5000c5004f0e566d'};
+  % my $hostname = (split(/\./, $get_var->("SUT_IP")))[0];
+  % my $device_id = defined($wwn->{$hostname}) ? '/dev/disk/by-id/' . $wwn->{$hostname} : '';
+      <device><%= $device_id %></device>
```

Comment:
For SUTs other than lipa or briza, device_id will be empty. Is it okay?

Reply:
if mutil disks, it display '<device>/dev/disk/by-id/wwn-0x5002538c4002f8bd</device>'
if single disk, it display '<device></device>', the system will be installed on the available disk randomly. 
I think it is logical.

Reply:
ok